internal/saucecloud/cloud.go

```diff
@@ -336,6 +340,14 @@ func (r *CloudRunner) runJobs(jobOpts chan job.StartOptions, results chan<- resu
 			continue
 		}
 
+		// Attempt starts from 0 while MaxAttempts starts from 1. So here is comparing Attempt with MaxAttempts-1
+		if opts.Attempt < opts.MaxAttempts-1 && opts.CurrentPassCount < opts.PassCount {
+			opts.Attempt++
+			jobOpts <- opts
+			log.Info().Str("suite", opts.DisplayName).Str("attempt", fmt.Sprintf("%d of %d", opts.Attempt+1, opts.MaxAttempts)).Msg("Retrying suite.")
+			continue
+		}
+
```

Comment:
So what happens in this case following case?

I want to run 5 times, with a pass-threshold of 2. My suite fails 4 times and on the last run it passes? What's the final status/exitcode of saucectl?

Reply:
good point... updating